misago/users/utils.py

```diff
@@ -1,5 +1,15 @@
+from django.utils import timezone
+
+from misago.users.models.deleteduser import DeletedUser
+
 import hashlib
 
 
 def hash_email(email):
     return hashlib.md5(email.lower().encode('utf-8')).hexdigest()
+
+def create_user_deletion(deleted_by, deleted_on=timezone.now()):
```

Comment:
I think its better to have three functions taking no arguments at all:

- `record_user_deleted_by_self`
- `record_user_deleted_by_staff`
- `record_user_deleted_by_system`